pom.xml

```diff
@@ -702,74 +710,15 @@
     </build>
 
     <profiles>
-        <profile>
-            <id>jdk8</id>
-            <activation>
-                <jdk>[1.8,1.9)</jdk>
-            </activation>
-            <build>
-                <pluginManagement>
-                    <plugins>
-                        <plugin>
-                            <groupId>org.apache.maven.plugins</groupId>
-                            <artifactId>maven-toolchains-plugin</artifactId>
-                            <configuration>
-                                <toolchains>
-                                    <jdk>
-                                        <version>1.8</version>
-                                        <vendor>oracle</vendor>
-                                    </jdk>
-                                </toolchains>
-                            </configuration>
-                        </plugin>
-                    </plugins>
-                </pluginManagement>
-            </build>
-        </profile>
         <profile>
             <id>jdk9</id>
             <activation>
                 <jdk>[1.9,)</jdk>
```

Comment:
why leave this in though?

Reply:
These plugins currently don't run on jdk9+.  For example Spotbugs (I think) finds bogus bugs that only exist in java 11 generated code and can't be fixed in source.  :rage:

Reply:
If you changed the id name to something that says as much, it'd make me happy :p

Reply:
I don't want to just go changing profile names around.  People can refer to them, for example we did so for other profiles with `-Ptoolchains`.  We should focus our efforts on getting the build clean on all supported JDKs.